cfn-resources/stream-connection/mongodb-atlas-streamconnection.json

```diff
@@ -0,0 +1,173 @@
+{
+  "typeName": "MongoDB::Atlas::StreamConnection",
+  "additionalProperties": false,
+  "description": "Returns, adds, edits, and removes one connection for a stream instance in the specified project. To use this resource, the requesting API Key must have the Project Owner roles.",
```

Comment:
we are missing documentationUrl here

Reply:
added in the doc PR: https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/936